src/osd/OSD.cc

```diff
@@ -1660,6 +1660,13 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     } else {
       op_tracker.dump_ops_in_flight(f);
     }
+  } else if (command == "dump_blocked_ops") {
+    if (!op_tracker.tracking_enabled) {
+      ss << "op_tracker tracking is not enabled";
```

Comment:
I don't mind leaving in a switch for advanced users to disable the optracker (there is still overhead). I just want to think about some way of knowing that we have untracked ops still in the pipeline. I'm not sure if that's really feasible or not (without, well, tracking them).

One idea: could we keep a count of the in-flight ops even when the actual dynamic tracking is disabled (eg by incrementing a [possibly sharded] counter in the constructor and destructor)? Then if the user enables the optracker, we have a count of untracked ops which still need to complete, and we can include that information in the dumps ("6 untracked ops still in the system" or something).